packages/firestore/exp/index.ts

```diff
@@ -130,3 +131,5 @@ export { Timestamp } from '../src/exp/timestamp';
 export { CACHE_SIZE_UNLIMITED } from '../src/exp/database';
 
 export { FirestoreErrorCode, FirestoreError } from '../src/util/error';
+
+export { AbstractUserDataWriter } from '../src/lite/user_data_writer';
```

Comment:
Does this not break our public API generation? I would think that this ends up showing up in `public.d.ts` (but it should be an easy fix that I can provide as a follow up)

Reply:
It's removed from `public.d.ts` because it's `@internal.`